typings/telegram-types.d.ts

```diff
@@ -179,6 +179,32 @@ export type InputFile =
  */
 export type InputFileVideoNote = Exclude<InputFile, InputFileByURL>
 
+export interface AdminPerms {
```

Comment:
Why does it call `AdminPerms`? There isn't that type in bots API docs, where the same called `ChatPermissions`.

Even if Admin word is more relative to the context than Chat, anyway shortening is not a good when all other types called with their full names!

Reply:
You mean, you'd prefer `AdminPermissions`, or maybe `ExtraPromote`, for consistency with `telegram-types.d.ts`?

Reply:
https://core.telegram.org/bots/api#chatpermissions

Reply:
right.. its not ChatPermissions, its extra of [promoteChatMember](https://core.telegram.org/bots/api#promotechatmember)

so `ExtraPromoteChatMember` seems more fitting as @wojpawlik suggested